app/controllers/api/v1/applications_controller.rb

```diff
@@ -21,6 +23,14 @@ def show
       def application_params
         params.require(:application).permit(:title, :description, :domain)
       end
+
+      def authenticate!
+        render json: { error: 'Unathorized' }, status: 401 unless current_user
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.